### DISCUSSION TITLE
Gratis Leisure suit Larry 1, 2, 3, 5, 6 & 7

### DISCUSSION THREAD

**Discussion by jschwartzenberg** [dosemu2/dosemu2#1534] [General]
They're giving away the first two LSL games:
https://freebies.indiegala.com/leisure-suit-larry-1-in-the-land-of-the-lounge-lizards
https://freebies.indiegala.com/leisure-suit-larry-2-looking-for-love-in-several-wrong-places

In addition to a Windows build of ScummVM, I also see the DOS executables ;)

**Comment by jschwartzenberg** [comment]
They added the third one too!
https://freebies.indiegala.com/leisure-suit-larry-3-passionate-patti-in-pursuit-of-the-pulsating-pectorals

**Reply by jschwartzenberg** [reply]
Today again the third one for free:
https://freebies.indiegala.com/leisure-suit-larry-3-passionate-patti-in-pursuit-of-the-pulsating-pectorals

**Comment by jschwartzenberg** [comment]
And the fifth!
https://freebies.indiegala.com/leisure-suit-larry-5-passionate-patti-does-a-little-undercover-work

**Comment by jschwartzenberg** [comment]
And the sixth one!
https://freebies.indiegala.com/leisure-suit-larry-6-shape-up-or-slip-out

With working DOS binaries!

**Reply by jschwartzenberg** [reply]
Today again: https://freebies.indiegala.com/leisure-suit-larry-6-shape-up-or-slip-out

**Comment by jschwartzenberg** [comment]
They even added seven!
https://freebies.indiegala.com/leisure-suit-larry-7-love-for-sail

Haven't checked the DOS binaries yet, but there's a DOSBox build for Windows included, so supposedly it would run in there.

**Reply by jschwartzenberg** [reply]
And 7 again now: https://freebies.indiegala.com/leisure-suit-larry-7-love-for-sail

**Comment by stsp** [comment]
The only remaining thing
is to release sources. :)

**Reply by jschwartzenberg** [reply]
Yep, those would be the sources to various versions of Sierra's Creative Interpreter (SCI). It would be interesting if those are still around somewhere. I believe ScummVM has quite an accurate reimplementation.

**Reply by jschwartzenberg** [reply]
And actually the AGI interpreter used by the first game.

Also note that they did not release the LSL1 remake and the LSL6 SVGA version in this series.
